snapshot/lib/snapshot/test_command_generator.rb

```diff
@@ -33,8 +33,9 @@ def pipe(log_path: nil)
 
         xcpretty = "xcpretty #{Snapshot.config[:xcpretty_args]}"
         xcpretty << "--no-color" if Helper.colors_disabled?
-
-        return pipe << "| #{xcpretty}"
+        pipe << "| #{xcpretty}"
+        pipe << " > /dev/null" if Snapshot.config[:suppress_xcode_output]
```

Comment:
```suggestion
        pipe << "> /dev/null" if Snapshot.config[:suppress_xcode_output]
```

😱 My bad! I didn't realize this pipe was going into an array. This array later on gets a `.join(' ')` which adds the space. Sorry about that!

Reply:
Oh, I missed that. Thanks for catching it!